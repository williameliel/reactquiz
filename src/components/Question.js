import React from 'react';

import Choices from './Choices';

function Question(props){

    if (!props.questions) {
        return false;
    }

    return (

      <div>
         <h2>{props.questions[props.current_step].question}</h2>
         <Choices 
           questionId={props.current_step}
           onChoiceSelect={props.onChoiceSelect}
           choices={props.questions[props.current_step].choices} />
      </div>

    );

}

export default Question;
