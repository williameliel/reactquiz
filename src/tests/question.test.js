import React from 'react';
import Question from '../components/Question';
import renderer from 'react-test-renderer';

it('Question renders correctly', () => {
  const tree = renderer.create(
    <Question />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
