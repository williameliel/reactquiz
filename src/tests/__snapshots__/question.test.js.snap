exports[`test Question renders correctly 1`] = `null`;
